pkg/kv/kvserver/consistency_queue_test.go

```diff
@@ -234,46 +241,64 @@ func TestCheckConsistencyInconsistent(t *testing.T) {
 				t.Errorf("diff = %v", d)
 			}
 
-			diff[0].Timestamp.Logical = 987 // mock this out for a consistent string below
-
-			act := diff.String()
-
-			exp := `--- leaseholder
-+++ follower
-+0.000000123,987 "e"
-+    ts:1970-01-01 00:00:00.000000123 +0000 UTC
-+    value:"\x00\x00\x00\x00\x01T"
-+    raw mvcc_key/value: 6500000000000000007b000003db0d 000000000154
-`
-			if act != exp {
-				// We already logged the actual one above.
-				t.Errorf("expected:\n%s\ngot:\n%s", exp, act)
-			}
-
+			// With a real clock its not possible to check the actual diff,
+			// so we settle for just a key check.
```

Comment:
Why can't you add `diff[0].Timestamp.WallTime=123` to the old code?

Reply:
My understanding is that the timestamp is part of the raw MVCC record, which makes this fairly hard to diff. Here is an example with 2 consecutive runs of the test with this code restored to as it. 

`    consistency_queue_test.go:257: expected:
        --- leaseholder
        +++ follower
        +0.000000123,987 "e"
        +    ts:1970-01-01 00:00:00.000000123 +0000 UTC
        +    value:"\x00\x00\x00\x00\x01T"
        +    raw mvcc_key/value: 6500000000000000007b000003db0d 000000000154
        
        got:
        --- leaseholder
        +++ follower
        +1593109825.736774000,987 "e"
        +    ts:2020-06-25 18:30:25.736774 +0000 UTC
        +    value:"\x00\x00\x00\x00\x01T"
        +    raw mvcc_key/value: 6500161bdcf223399170000003db0d 000000000154


            consistency_queue_test.go:257: expected:
        --- leaseholder
        +++ follower
        +0.000000123,987 "e"
        +    ts:1970-01-01 00:00:00.000000123 +0000 UTC
        +    value:"\x00\x00\x00\x00\x01T"
        +    raw mvcc_key/value: 6500000000000000007b000003db0d 000000000154
        
        got:
        --- leaseholder
        +++ follower
        +1593109975.875626000,987 "e"
        +    ts:2020-06-25 18:32:55.875626 +0000 UTC
        +    value:"\x00\x00\x00\x00\x01T"
        +    raw mvcc_key/value: 6500161bdd151832a410000003db0d 000000000154`

Reply:
Specifically note: `raw mvcc_key/value: 6500161bdcf223399170000003db0d 000000000154` vs `raw mvcc_key/value: 6500161bdd151832a410000003db0d 000000000154`

Reply:
Sorry, this dropped off my radar. I would "just" regexp-replace `s~(raw mvcc_key/value: ).*~\1 <omitted>~g` so that we compare the parts we know should match.

Reply:
I think in that case the only thing that is comparable is the value, since everything else has a timestamp, so we can just compare those directly without using strings.

Reply:
hmm, I just overrode the timestamp and that seemed to have solved it all. Ok going with that.